File: AULA_02/C/AnoBissexto.c
#include <stdio.h>

int main(){

  int ano;
  printf("Qual é o ano? ");
  scanf("%d",&ano);

  if ( !(ano%400) || (!(ano%4) && (ano%100)) ){
    printf("É bissexto\n");
  }else{
    printf("Não é bissexto\n");
  }

  return 0;
}

File: AULA_02/Haskell/AnoBissexto.hs
main = do
       putStrLn "Qual ano? "
       line <- getLine
       let ano = read line :: Int 
       let bis = ((ano `rem` 400) == 0) || ((ano `rem` 4) == 0) && ((ano `rem` 100) /= 0)
       putStrLn $ show bis 
